packages/data/src/components/use-select/test/suspense.js

```diff
@@ -157,7 +157,7 @@ describe( 'useSuspenseSelect', () => {
 		);
 
 		const rendered = render( <App /> );
-		const label = await waitFor( () => rendered.getByLabelText( 'error' ) );
+		const label = await rendered.findByLabelText( 'error' );
```

Comment:
Shouldn't we use `screen.findByLabelText` ?

Reply:
This is a [different rule](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/render-result-naming-convention.md), though. 😉 One at a time, please.

Reply:
Woops I got my reviews mixed up. Nevermind!